### DISCUSSION TITLE
Show custom domain somewhere in editor

### DISCUSSION THREAD

**Discussion by StachuDotNet** [darklang/dark#4317] [Feedback]
e.g. on https://darklang.com/a/ops-presence, I'd like to see https://presence.darklang.com somewhere.

**Comment by StachuDotNet** [comment]
This was a matter of discover-ability for me - I had access to the canvas, but had to search through Dark code to find the corresponding URL.

It's unlikely anyone would forget their single custom domain, but in an org setting it may matter more.

**Comment by pbiggar** [comment]
we could load it in intial_load, it's an easy DB lookup.

Though I want to move away from initial_load being the be-all and end-all to get everything, so maybe we could make a request to an API for this on the dark-editor canvas.

**Comment by StachuDotNet** [comment]
Sounds great as an endpoint in dark-editor!

**Comment by StachuDotNet** [comment]
Closing #3696 in favor of this.
From that Issue:

> Users can click on the `...` menu in handlers to get a link to the handler, but we don't have that link presented anywhere more obvious. We should add a link, maybe in the menu?

**Comment by StachuDotNet** [comment]
Especially as this relates to darklang.io domains eventually being used, and allowing _multiple_ domains for a canvas (routing different routes to different domains), this is a bit trickier and should be spelled out before being an Issue, so migrated to a Discussion here.
